### DISCUSSION TITLE
Needhelp: Firebase auth emulator not working as expected

### DISCUSSION THREAD

**Discussion by prakashchokalingam** [nuxt-community/firebase-module#456] [Q&A] [answered]
The firebase auth emulator alone not working properly,

Versions:
"nuxt": "2.14.9",
"@nuxtjs/firebase": "^7.1.2",

Node: 
"node": ">= 12.13.0"

Firebase nuxt configurations,
```
['@nuxtjs/firebase', {
      config: {
        projectId: "####,
        apiKey: "####",
      },
      services: {
        firestore: {
          emulatorPort: 8082
        },
        auth: {
          initialize: {
            onAuthStateChangedAction: "manageAuthState",
          },
          emulatorHost: 'http://localhost',
          emulatorPort: 9099,
        },
        functions: {
          emulatorPort: 5001,
        }
      }
    }],
```

And the emulators are running successfully on the specified port,

<img width="592" alt="Screenshot 2021-02-11 at 9 36 12 AM" src="https://user-images.githubusercontent.com/5512765/107601408-a961b800-6c4c-11eb-8351-74c700530365.png">

Yet getting the below warning,

```
 Received a signed JWT. Auth Emulator does not validate JWTs and IS NOT SECURE
```

<img width="717" alt="Screenshot 2021-02-11 at 9 39 32 AM" src="https://user-images.githubusercontent.com/5512765/107601539-0b222200-6c4d-11eb-8327-a49a0f59f962.png">

Missing anything else? other emulators like firestore and all working fine.

**Comment by lupas** [comment, marked as answer]
We're working on improving auth SSR working with the emulator in https://github.com/nuxt-community/firebase-module/issues/465 and https://github.com/nuxt-community/firebase-module/issues/465. One thing you still have to do is to set the FIREBASE_AUTH_EMULATOR_HOST env variable to your localhost.

Follow https://github.com/nuxt-community/firebase-module/issues/465 for us trying to resolve this issue, I'm sure we'll find a solution soon.
